src/main/resources/templates/subscriptions.html

```diff
@@ -293,7 +297,7 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                     Jenkins: As a jenkins user it is recommended that you set this as an API token and not user password. Not to be mistaken for the job token that is set for a single job." />
                             <div>
                                 <input id="passwordInput" title="Enter password"
-                                    data-bind="visible: $root.showPassword() && ($root.mode() == 'edit' || $root.mode() == 'add'),textInput:$data.password"
+                                    data-bind="visible: $root.showPassword() && ($root.mode() == 'edit' || $root.mode() == 'add' || $root.mode() == 'clone'),textInput:$data.password"
```

Comment:
These three comparisons could be written shorter, like this:
$root.mode.match(/^(edit|add|clone)$/)
It's a matter of taste though I guess.

Reply:
Done